### PR TITLE
Feature/ocean contracts

### PR DESCRIPTION
**Pull request by ssallam**
* run ocean-contracts docker to deploy contracts
* update aquarius inputs as required by the decentralized ddo version

### CODE REVIEW

File: start_ocean.sh [hunk 1]
```diff
@@ -42,9 +46,16 @@ else
...
+# Use this seed on ganache to always create the same wallets
 export GANACHE_MNEMONIC=${GANACHE_MNEMONIC:-"taxi music thumb unique chat sand crew more leg another off lamp"}
 
+# Ocean contracts
+export OCEAN_HOME="${HOME}/.ocean"
+export CONTRACTS_OWNER_ROLE_ADDRESS="${CONTRACTS_OWNER_ROLE_ADDRESS}"
+export DEPLOY_CONTRACTS="true"
+export OCEAN_ARTIFACTS_FOLDER="${OCEAN_HOME}/ocean-contracts/artifacts"
+# Specify which ethereum client to run or connect to: development
+export CONTRACTS_NETWORK_NAME="ganache"
```
Comment (alexcos20) [inline start_ocean.sh:58]:
```suggestion
export CONTRACTS_VERSION="test"
```

**Comment by kremalicious** [comment]
`ocean-contracts` needs to be added to main README, in the table and as a building block

File: start_ocean.sh [hunk 2]
```diff
@@ -27,12 +27,16 @@ DIR="${DIR/ /\\ }"
 COMPOSE_DIR="${DIR}/compose-files"
 
 # Default versions of Aquarius, Provider
-export AQUARIUS_VERSION=${AQUARIUS_VERSION:-v3}
+export AQUARIUS_VERSION=${AQUARIUS_VERSION:-latest}
```
Comment (alexcos20) [inline start_ocean.sh:30]:
```suggestion
export AQUARIUS_VERSION=${AQUARIUS_VERSION:-decentralized}
```

Comment (alexcos20) [reply start_ocean.sh:30]: @ssallam

**Pull request merged**